ldap/servers/plugins/replication/cl5_api.c

```diff
@@ -2787,8 +2787,8 @@ _cl5TrimEntry(dbi_val_t *key, dbi_val_t *data, void *ctx)
             dblcictx->finished = PR_TRUE;
             return DBI_RC_NOTFOUND;
         } else {
-            slapi_log_err(SLAPI_LOG_REPL, repl_plugin_name_cl,
-                          "_cl5TrimReplica - Changelog purge skipped anchor csn %s\n",
+            slapi_log_err(SLAPI_LOG_ERR, repl_plugin_name_cl,
+                          "_cl5TrimEntry - Changelog purge skipped anchor csn %s\n",
```

Comment:
I don't think we should log this by default as it could be normal to skip the anchor csn